packages/airplane/internal/config.ts

```diff
@@ -520,3 +530,146 @@ const normalizeConfig = <TParams extends InputParams>(
     name: config.name ? config.name : humanizeString(config.slug),
   };
 };
+
+/**
+ * The configuration for a SQL Task. To learn more, see the
+ * [Task definition reference](https://docs.airplane.dev/tasks/task-definition).
+ *
+ * Builtin task bindings are currently in beta.
+ */
+export type SQLTaskConfig<TParams extends InputParams> = TaskConfig<TParams> &
```

Comment:
This is bringing back the omitted fields, e.g. `resources`:

<img width="839" alt="CleanShot 2023-04-05 at 15 50 32" src="https://user-images.githubusercontent.com/2907397/230190967-2bfb52ea-642a-4b1f-885c-62d8c19f4292.png">

```suggestion
export type SQLTaskConfig<TParams extends InputParams> =
```